main.go

```diff
@@ -557,6 +578,23 @@ func migrateDB(ctx context.Context, sqlDB *sql.DB, dbName, dbAddress, dbUsername
 }
 
 func main() {
+	if _, err := os.Stat(".env"); err == nil {
+		log.Println("Overriding ENV with .env")
+		err := godotenv.Load()
+		if err != nil {
+			log.Fatalf("Error loading .env file: %s", err)
+		}
+	}
+
+	if v, err := strconv.ParseBool(os.Getenv("PERCONA_TEST_WAIT_FOR_DEBUG_SESSION")); err == nil && v {
+		shouldWait := true // will be overridden via debug session
+		//goland:noinspection ALL
+		for shouldWait {
+			log.Println("Waiting for debugging session...")
+			time.Sleep(time.Second * 1)
+		}
+	}
```

Comment:
when `PERCONA_TEST_WAIT_FOR_DEBUG_SESSION=true`, you can connect with intellij idea and override shouldWait=false

this way you can debug in IDE from the begging on the main